profiles/common/modules/features/nexteuropa_editorial/includes/nexteuropa_editorial.tracking.inc

```diff
@@ -0,0 +1,429 @@
+<?php
+/**
+ * @file
+ * Code specific to customization for the change tracking management.
+ */
+
+/**
+ * Field widget form alter callback.
+ *
+ * It inject a specific validate callback in achrge of scanning field value.
+ *
+ * @see nexteuropa_editorial_field_widget_text_textarea_form_alter()
+ * @see nexteuropa_editorial_field_widget_text_textarea_with_summary_form_alter()
+ * @see nexteuropa_editorial_field_widget_text_textfield_form_alter()
+ */
+function _nexteuropa_editorial_field_widget_tracking_form_alter(&$element, &$form_state, $context) {
+  // So far, only nodes are supported.
```

Comment:
If so in following lines there should be checkpoint for 'node' type otherwise this function will be triggered all the time.